### ISSUE TITLE
Mac Java 11 NettyNioAsyncHttpClientSpiVerificationTest Failed

### ISSUE THREAD

**Issue by bigapple10027** [aws/aws-sdk-java-v2#1038]
System config:
```
MacOS High Seerria  10.13.6
Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T11:41:47-07:00)

openjdk version "11.0.2" 2019-01-15
OpenJDK Runtime Environment 18.9 (build 11.0.2+9)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+9, mixed mode)
```

When installing the SDK with maven, I got the following test failure:

```java
[INFO] Running software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClientSpiVerificationTest
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.23 s <<< FAILURE! - in software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClientSpiVerificationTest
[ERROR] signalsErrorViaOnErrorAndFuture(software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClientSpiVerificationTest)  Time elapsed: 2.113 s  <<< ERROR!
java.util.concurrent.TimeoutException
	at software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClientSpiVerificationTest.signalsErrorViaOnErrorAndFuture(NettyNioAsyncHttpClientSpiVerificationTest.java:93)
```
Currently I'm trying to use `mvn clean install -P quick` to bypass the test failure. Can I get some help on finding the root cause of the failure and fix the test?

**Comment by dagnir** [comment]
Hi @bigapple10027 the test creates a Wiremock server and attempts to talk to it at `http://localhost + mockServer.port()`. The test can timeout if it it's having trouble finding `localhost`. Please check your system settings to ensure `localhost` resolves to your local machine.

**Comment by patrick-tripp** [comment]
MacOS Mojave 10.14.3

I'm having the exact same issue. I tried disabling my firewall, that didn't fix it. I can connect to my localhost webserver with no issues.

**Comment by ikester** [comment]
I'm getting the same error when trying to build `master` (`commit 9c0ce9cde15e7f5bbceaeccf64030d5af3b3fc63`) in High Sierra. With both the openJDK and the Oracle LTS JDKs.

```
java 11.0.2 2019-01-15 LTS
Java(TM) SE Runtime Environment 18.9 (build 11.0.2+9-LTS)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.2+9-LTS, mixed mode)

Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T14:41:47-04:00)
Maven home: /Users/xxxx/java/apache-maven-3.6.0
Java version: 11.0.2, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-11.0.2.jdk/Contents/Home
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.13.6", arch: "x86_64", family: "mac"
```

**Comment by Kailash2509** [comment]
I am also getting same error 

signalsErrorViaOnErrorAndFuture(software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClientSpiVerificationTest)  Time elapsed: 1.125 s  <<< ERROR!
java.util.concurrent.TimeoutException
	at software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClientSpiVerificationTest.signalsErrorViaOnErrorAndFuture(NettyNioAsyncHttpClientSpiVerificationTest.java:91)

**Comment by caoliancai** [comment]
I am having the same issue:
"
[ERROR] signalsErrorViaOnErrorAndFuture(software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClientSpiVerificationTest)  Time elapsed: 1.059 s  <<< ERROR!
java.util.concurrent.TimeoutException
        at software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClientSpiVerificationTest.signalsErrorViaOnErrorAndFuture(NettyNioAsyncHttpClientSpiVerificationTest.java:91)

[INFO] Running software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClientWireMockTest
[INFO] Tests run: 18, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 26.005 s - in software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClientWireMockTest
[INFO] Running software.amazon.awssdk.http.nio.netty.SdkEventLoopGroupTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.013 s - in software.amazon.awssdk.http.nio.netty.SdkEventLoopGroupTest
[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   NettyNioAsyncHttpClientSpiVerificationTest.signalsErrorViaOnErrorAndFuture:91 ? Timeout
[INFO]
[ERROR] Tests run: 65, Failures: 0, Errors: 1, Skipped: 1
[INFO]
"
With Windows 10, JDK 8

**Comment by zoewangg** [comment]
Fixed via #1139 

@caoliancai Building SDK with Windows is tracked in #986
